2022-07-09 12:46:30 error: [PUT] /api/v1/user/33 >> StatusCode:: 400, Message:: email must be an email
2022-07-09 14:08:32 error: [PUT] /api/v1/user/33 >> StatusCode:: 400, Message:: email must be an email
2022-07-09 14:08:59 error: [PUT] /api/v1/user/33 >> StatusCode:: 400, Message:: email must be an email
2022-07-09 14:09:30 error: [PUT] /api/v1/user/33 >> StatusCode:: 400, Message:: email must be an email
2022-07-09 14:09:49 error: [PUT] /api/v1/user/33 >> StatusCode:: 400, Message:: email must be an email
2022-07-09 14:10:17 error: [PUT] /api/v1/user/33 >> StatusCode:: 400, Message:: email must be an email
2022-07-09 14:13:52 error: [PUT] /api/v1/user/33 >> StatusCode:: 400, Message:: email must be an email
2022-07-09 14:13:53 error: [PUT] /api/v1/user/33 >> StatusCode:: 400, Message:: email must be an email
2022-07-09 14:14:06 error: [PUT] /api/v1/user/33 >> StatusCode:: 400, Message:: email must be an email
2022-07-09 14:16:01 error: [PUT] /api/v1/user/33 >> StatusCode:: 400, Message:: email must be an email
2022-07-09 14:18:07 error: [POST] /api/v1/user >> StatusCode:: 400, Message:: Unexpected token } in JSON at position 52
2022-07-09 14:18:13 error: [POST] /api/v1/user >> StatusCode:: 500, Message:: Value for User.email is required, 'undefined' found
entity: User {
  createdAt: 2022-07-09T21:18:13.574Z,
  updatedAt: 2022-07-09T21:18:13.574Z,
  firstName: 'Gage',
  lastName: 'Tylee'
}
2022-07-09 14:19:27 error: [POST] /api/v1/user >> StatusCode:: 500, Message:: Value for User.lastName is required, 'undefined' found
entity: User {
  createdAt: 2022-07-09T21:19:27.818Z,
  updatedAt: 2022-07-09T21:19:27.818Z,
  firstName: 'Gage',
  email: 'hello@gmail.com'
}
2022-07-09 14:21:31 error: uncaughtException: write EPIPE
Error: write EPIPE
    at process.target._send (node:internal/child_process:844:20)
    at process.target.send (node:internal/child_process:717:19)
    at Object.exports.send (/Users/gagetylee/callio-api/node_modules/ts-node-dev/lib/ipc.js:17:14)
    at /Users/gagetylee/callio-api/node_modules/ts-node-dev/lib/wrap.js:88:9
    at Object.nodeDevHook [as .js] (/Users/gagetylee/callio-api/node_modules/ts-node-dev/lib/hook.js:61:17)
    at Module.load (node:internal/modules/cjs/loader:975:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Module.require (node:internal/modules/cjs/loader:999:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (/Users/gagetylee/callio-api/node_modules/lodash/_createAggregator.js:1:23)
2022-07-09 14:21:31 error: uncaughtException: write EPIPE
Error: write EPIPE
    at process.target._send (node:internal/child_process:844:20)
    at process.target.send (node:internal/child_process:717:19)
    at Object.exports.send (/Users/gagetylee/callio-api/node_modules/ts-node-dev/lib/ipc.js:17:14)
    at /Users/gagetylee/callio-api/node_modules/ts-node-dev/lib/wrap.js:88:9
    at Object.nodeDevHook [as .js] (/Users/gagetylee/callio-api/node_modules/ts-node-dev/lib/hook.js:61:17)
    at Module.load (node:internal/modules/cjs/loader:975:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Module.require (node:internal/modules/cjs/loader:999:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (/Users/gagetylee/callio-api/node_modules/knex/lib/query/querycompiler.js:11:17)
2022-07-09 14:21:31 error: uncaughtException: write EPIPE
Error: write EPIPE
    at process.target._send (node:internal/child_process:844:20)
    at process.target.send (node:internal/child_process:717:19)
    at Object.exports.send (/Users/gagetylee/callio-api/node_modules/ts-node-dev/lib/ipc.js:17:14)
    at /Users/gagetylee/callio-api/node_modules/ts-node-dev/lib/wrap.js:88:9
    at Object.nodeDevHook [as .js] (/Users/gagetylee/callio-api/node_modules/ts-node-dev/lib/hook.js:61:17)
    at Module.load (node:internal/modules/cjs/loader:975:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Module.require (node:internal/modules/cjs/loader:999:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (/Users/gagetylee/callio-api/node_modules/lodash/groupBy.js:2:24)
2022-07-09 14:21:31 error: uncaughtException: write EPIPE
Error: write EPIPE
    at process.target._send (node:internal/child_process:844:20)
    at process.target.send (node:internal/child_process:717:19)
    at Object.exports.send (/Users/gagetylee/callio-api/node_modules/ts-node-dev/lib/ipc.js:17:14)
    at /Users/gagetylee/callio-api/node_modules/ts-node-dev/lib/wrap.js:88:9
    at Object.nodeDevHook [as .js] (/Users/gagetylee/callio-api/node_modules/ts-node-dev/lib/hook.js:61:17)
    at Module.load (node:internal/modules/cjs/loader:975:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Module.require (node:internal/modules/cjs/loader:999:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (/Users/gagetylee/callio-api/node_modules/lodash/_createAggregator.js:2:22)
2022-07-09 14:21:31 error: uncaughtException: write EPIPE
Error: write EPIPE
    at process.target._send (node:internal/child_process:844:20)
    at process.target.send (node:internal/child_process:717:19)
    at Object.exports.send (/Users/gagetylee/callio-api/node_modules/ts-node-dev/lib/ipc.js:17:14)
    at /Users/gagetylee/callio-api/node_modules/ts-node-dev/lib/wrap.js:88:9
    at Object.nodeDevHook [as .js] (/Users/gagetylee/callio-api/node_modules/ts-node-dev/lib/hook.js:61:17)
    at Module.load (node:internal/modules/cjs/loader:975:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Module.require (node:internal/modules/cjs/loader:999:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (/Users/gagetylee/callio-api/node_modules/knex/lib/query/querycompiler.js:12:13)
2022-07-09 14:21:31 error: uncaughtException: write EPIPE
Error: write EPIPE
    at process.target._send (node:internal/child_process:844:20)
    at process.target.send (node:internal/child_process:717:19)
    at Object.exports.send (/Users/gagetylee/callio-api/node_modules/ts-node-dev/lib/ipc.js:17:14)
    at /Users/gagetylee/callio-api/node_modules/ts-node-dev/lib/wrap.js:88:9
    at Object.nodeDevHook [as .js] (/Users/gagetylee/callio-api/node_modules/ts-node-dev/lib/hook.js:61:17)
    at Module.load (node:internal/modules/cjs/loader:975:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Module.require (node:internal/modules/cjs/loader:999:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (/Users/gagetylee/callio-api/node_modules/lodash/has.js:1:15)
2022-07-09 14:21:31 error: uncaughtException: write EPIPE
Error: write EPIPE
    at process.target._send (node:internal/child_process:844:20)
    at process.target.send (node:internal/child_process:717:19)
    at Object.exports.send (/Users/gagetylee/callio-api/node_modules/ts-node-dev/lib/ipc.js:17:14)
    at /Users/gagetylee/callio-api/node_modules/ts-node-dev/lib/wrap.js:88:9
    at Object.nodeDevHook [as .js] (/Users/gagetylee/callio-api/node_modules/ts-node-dev/lib/hook.js:61:17)
    at Module.load (node:internal/modules/cjs/loader:975:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Module.require (node:internal/modules/cjs/loader:999:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (/Users/gagetylee/callio-api/node_modules/knex/lib/query/querycompiler.js:14:13)
2022-07-09 14:21:31 error: uncaughtException: write EPIPE
Error: write EPIPE
    at process.target._send (node:internal/child_process:844:20)
    at process.target.send (node:internal/child_process:717:19)
    at Object.exports.send (/Users/gagetylee/callio-api/node_modules/ts-node-dev/lib/ipc.js:17:14)
    at /Users/gagetylee/callio-api/node_modules/ts-node-dev/lib/wrap.js:88:9
    at Object.nodeDevHook [as .js] (/Users/gagetylee/callio-api/node_modules/ts-node-dev/lib/hook.js:61:17)
    at Module.load (node:internal/modules/cjs/loader:975:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Module.require (node:internal/modules/cjs/loader:999:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (/Users/gagetylee/callio-api/node_modules/knex/lib/query/querycompiler.js:15:16)
2022-07-09 14:21:31 error: uncaughtException: write EPIPE
Error: write EPIPE
    at process.target._send (node:internal/child_process:844:20)
    at process.target.send (node:internal/child_process:717:19)
    at Object.exports.send (/Users/gagetylee/callio-api/node_modules/ts-node-dev/lib/ipc.js:17:14)
    at /Users/gagetylee/callio-api/node_modules/ts-node-dev/lib/wrap.js:88:9
    at Object.nodeDevHook [as .js] (/Users/gagetylee/callio-api/node_modules/ts-node-dev/lib/hook.js:61:17)
    at Module.load (node:internal/modules/cjs/loader:975:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Module.require (node:internal/modules/cjs/loader:999:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (/Users/gagetylee/callio-api/node_modules/lodash/omitBy.js:3:14)
2022-07-09 14:21:31 error: uncaughtException: write EPIPE
Error: write EPIPE
    at process.target._send (node:internal/child_process:844:20)
    at process.target.send (node:internal/child_process:717:19)
    at Object.exports.send (/Users/gagetylee/callio-api/node_modules/ts-node-dev/lib/ipc.js:17:14)
    at /Users/gagetylee/callio-api/node_modules/ts-node-dev/lib/wrap.js:88:9
    at Object.nodeDevHook [as .js] (/Users/gagetylee/callio-api/node_modules/ts-node-dev/lib/hook.js:61:17)
    at Module.load (node:internal/modules/cjs/loader:975:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Module.require (node:internal/modules/cjs/loader:999:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (/Users/gagetylee/callio-api/node_modules/lodash/pickBy.js:3:18)
2022-07-09 14:21:31 error: uncaughtException: write EPIPE
Error: write EPIPE
    at process.target._send (node:internal/child_process:844:20)
    at process.target.send (node:internal/child_process:717:19)
    at Object.exports.send (/Users/gagetylee/callio-api/node_modules/ts-node-dev/lib/ipc.js:17:14)
    at /Users/gagetylee/callio-api/node_modules/ts-node-dev/lib/wrap.js:88:9
    at Object.nodeDevHook [as .js] (/Users/gagetylee/callio-api/node_modules/ts-node-dev/lib/hook.js:61:17)
    at Module.load (node:internal/modules/cjs/loader:975:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Module.require (node:internal/modules/cjs/loader:999:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (/Users/gagetylee/callio-api/node_modules/lodash/_basePickBy.js:2:15)
2022-07-09 14:21:31 error: uncaughtException: write EPIPE
Error: write EPIPE
    at process.target._send (node:internal/child_process:844:20)
    at process.target.send (node:internal/child_process:717:19)
    at Object.exports.send (/Users/gagetylee/callio-api/node_modules/ts-node-dev/lib/ipc.js:17:14)
    at /Users/gagetylee/callio-api/node_modules/ts-node-dev/lib/wrap.js:88:9
    at Object.nodeDevHook [as .js] (/Users/gagetylee/callio-api/node_modules/ts-node-dev/lib/hook.js:61:17)
    at Module.load (node:internal/modules/cjs/loader:975:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Module.require (node:internal/modules/cjs/loader:999:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (/Users/gagetylee/callio-api/node_modules/knex/lib/client.js:17:23)
2022-07-09 14:21:31 error: uncaughtException: write EPIPE
Error: write EPIPE
    at process.target._send (node:internal/child_process:844:20)
    at process.target.send (node:internal/child_process:717:19)
    at Object.exports.send (/Users/gagetylee/callio-api/node_modules/ts-node-dev/lib/ipc.js:17:14)
    at /Users/gagetylee/callio-api/node_modules/ts-node-dev/lib/wrap.js:88:9
    at Object.nodeDevHook [as .js] (/Users/gagetylee/callio-api/node_modules/ts-node-dev/lib/hook.js:61:17)
    at Module.load (node:internal/modules/cjs/loader:975:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Module.require (node:internal/modules/cjs/loader:999:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (/Users/gagetylee/callio-api/node_modules/knex/lib/client.js:18:24)
2022-07-09 14:21:31 error: uncaughtException: write EPIPE
Error: write EPIPE
    at process.target._send (node:internal/child_process:844:20)
    at process.target.send (node:internal/child_process:717:19)
    at Object.exports.send (/Users/gagetylee/callio-api/node_modules/ts-node-dev/lib/ipc.js:17:14)
    at /Users/gagetylee/callio-api/node_modules/ts-node-dev/lib/wrap.js:88:9
    at Object.nodeDevHook [as .js] (/Users/gagetylee/callio-api/node_modules/ts-node-dev/lib/hook.js:61:17)
    at Module.load (node:internal/modules/cjs/loader:975:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Module.require (node:internal/modules/cjs/loader:999:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (/Users/gagetylee/callio-api/node_modules/knex/lib/schema/compiler.js:5:5)
2022-07-09 14:21:31 error: uncaughtException: write EPIPE
Error: write EPIPE
    at process.target._send (node:internal/child_process:844:20)
    at process.target.send (node:internal/child_process:717:19)
    at Object.exports.send (/Users/gagetylee/callio-api/node_modules/ts-node-dev/lib/ipc.js:17:14)
    at /Users/gagetylee/callio-api/node_modules/ts-node-dev/lib/wrap.js:88:9
    at Object.nodeDevHook [as .js] (/Users/gagetylee/callio-api/node_modules/ts-node-dev/lib/hook.js:61:17)
    at Module.load (node:internal/modules/cjs/loader:975:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Module.require (node:internal/modules/cjs/loader:999:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (/Users/gagetylee/callio-api/node_modules/knex/lib/client.js:19:22)
2022-07-09 14:31:18 error: [PUT] /api/v1/user/33 >> StatusCode:: 400, Message:: email must be an email
2022-07-09 14:31:55 error: [PUT] /api/v1/user/33 >> StatusCode:: 400, Message:: email must be an email
2022-07-09 14:32:33 error: [PUT] /api/v1/user/33 >> StatusCode:: 400, Message:: email must be an email
2022-07-09 14:36:40 error: [PUT] /api/v1/user/33 >> StatusCode:: 400, Message:: property hrs should not exist
2022-07-09 14:39:13 error: [PUT] /api/v1/user/33 >> StatusCode:: 400, Message:: email must be an email
2022-07-09 14:54:32 error: [POST] /api/v1/user >> StatusCode:: 400, Message:: lastName must be a string
2022-07-09 14:56:09 error: [PUT] /api/v1/user/36 >> StatusCode:: 404, Message:: User not found
2022-07-09 17:54:49 error: uncaughtException: Entity 'Collection' was not discovered, please make sure to provide it in 'entities' array when initializing the ORM (used in User.projects)
MetadataError: Entity 'Collection' was not discovered, please make sure to provide it in 'entities' array when initializing the ORM (used in User.projects)
    at Function.fromUnknownEntity (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/errors.js:148:16)
    at /Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/MetadataValidator.js:55:46
    at Array.forEach (<anonymous>)
    at /Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/MetadataValidator.js:53:67
    at Array.forEach (<anonymous>)
    at MetadataValidator.validateDiscovered (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/MetadataValidator.js:53:20)
    at MetadataDiscovery.findEntities (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/MetadataDiscovery.js:92:24)
    at MetadataDiscovery.discover (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/MetadataDiscovery.js:34:9)
    at MikroORM.discoverEntities (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/MikroORM.js:90:25)
    at Function.init (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/MikroORM.js:45:9)
2022-07-09 17:55:19 error: uncaughtException: Entity 'Collection' was not discovered, please make sure to provide it in 'entities' array when initializing the ORM (used in User.projects)
MetadataError: Entity 'Collection' was not discovered, please make sure to provide it in 'entities' array when initializing the ORM (used in User.projects)
    at Function.fromUnknownEntity (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/errors.js:148:16)
    at /Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/MetadataValidator.js:55:46
    at Array.forEach (<anonymous>)
    at /Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/MetadataValidator.js:53:67
    at Array.forEach (<anonymous>)
    at MetadataValidator.validateDiscovered (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/MetadataValidator.js:53:20)
    at MetadataDiscovery.findEntities (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/MetadataDiscovery.js:92:24)
    at MetadataDiscovery.discover (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/MetadataDiscovery.js:34:9)
    at MikroORM.discoverEntities (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/MikroORM.js:90:25)
    at Function.init (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/MikroORM.js:45:9)
2022-07-09 17:55:36 error: uncaughtException: User.projects has unknown 'inversedBy' reference: Collection.users
MetadataError: User.projects has unknown 'inversedBy' reference: Collection.users
    at Function.fromMessage (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/errors.js:172:16)
    at Function.fromWrongReference (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/errors.js:117:30)
    at MetadataValidator.validateOwningSide (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/MetadataValidator.js:82:42)
    at MetadataValidator.validateBidirectional (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/MetadataValidator.js:72:18)
    at MetadataValidator.validateEntityDefinition (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/MetadataValidator.js:33:18)
    at MetadataDiscovery.processEntity (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/MetadataDiscovery.js:354:24)
    at MetadataDiscovery.processDiscoveredEntities (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/MetadataDiscovery.js:76:43)
    at MetadataDiscovery.discover (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/MetadataDiscovery.js:35:9)
    at MikroORM.discoverEntities (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/MikroORM.js:90:25)
    at Function.init (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/MikroORM.js:45:9)
2022-07-09 17:57:42 error: uncaughtException: User.projects has unknown 'inversedBy' reference: Collection.users
MetadataError: User.projects has unknown 'inversedBy' reference: Collection.users
    at Function.fromMessage (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/errors.js:172:16)
    at Function.fromWrongReference (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/errors.js:117:30)
    at MetadataValidator.validateOwningSide (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/MetadataValidator.js:82:42)
    at MetadataValidator.validateBidirectional (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/MetadataValidator.js:72:18)
    at MetadataValidator.validateEntityDefinition (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/MetadataValidator.js:33:18)
    at MetadataDiscovery.processEntity (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/MetadataDiscovery.js:354:24)
    at MetadataDiscovery.processDiscoveredEntities (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/MetadataDiscovery.js:76:43)
    at MetadataDiscovery.discover (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/MetadataDiscovery.js:35:9)
    at MikroORM.discoverEntities (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/MikroORM.js:90:25)
    at Function.init (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/MikroORM.js:45:9)
2022-07-09 17:58:58 error: uncaughtException: User.projects has unknown 'inversedBy' reference: Collection.users
MetadataError: User.projects has unknown 'inversedBy' reference: Collection.users
    at Function.fromMessage (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/errors.js:172:16)
    at Function.fromWrongReference (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/errors.js:117:30)
    at MetadataValidator.validateOwningSide (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/MetadataValidator.js:82:42)
    at MetadataValidator.validateBidirectional (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/MetadataValidator.js:72:18)
    at MetadataValidator.validateEntityDefinition (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/MetadataValidator.js:33:18)
    at MetadataDiscovery.processEntity (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/MetadataDiscovery.js:354:24)
    at MetadataDiscovery.processDiscoveredEntities (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/MetadataDiscovery.js:76:43)
    at MetadataDiscovery.discover (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/MetadataDiscovery.js:35:9)
    at MikroORM.discoverEntities (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/MikroORM.js:90:25)
    at Function.init (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/MikroORM.js:45:9)
2022-07-09 18:02:04 error: uncaughtException: User.projects has unknown 'inversedBy' reference: Collection.users
MetadataError: User.projects has unknown 'inversedBy' reference: Collection.users
    at Function.fromMessage (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/errors.js:172:16)
    at Function.fromWrongReference (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/errors.js:117:30)
    at MetadataValidator.validateOwningSide (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/MetadataValidator.js:82:42)
    at MetadataValidator.validateBidirectional (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/MetadataValidator.js:72:18)
    at MetadataValidator.validateEntityDefinition (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/MetadataValidator.js:33:18)
    at MetadataDiscovery.processEntity (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/MetadataDiscovery.js:354:24)
    at MetadataDiscovery.processDiscoveredEntities (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/MetadataDiscovery.js:76:43)
    at MetadataDiscovery.discover (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/MetadataDiscovery.js:35:9)
    at MikroORM.discoverEntities (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/MikroORM.js:90:25)
    at Function.init (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/MikroORM.js:45:9)
2022-07-09 18:02:33 error: uncaughtException: Entity 'Collection' was not discovered, please make sure to provide it in 'entities' array when initializing the ORM (used in User.projects)
MetadataError: Entity 'Collection' was not discovered, please make sure to provide it in 'entities' array when initializing the ORM (used in User.projects)
    at Function.fromUnknownEntity (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/errors.js:148:16)
    at /Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/MetadataValidator.js:55:46
    at Array.forEach (<anonymous>)
    at /Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/MetadataValidator.js:53:67
    at Array.forEach (<anonymous>)
    at MetadataValidator.validateDiscovered (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/MetadataValidator.js:53:20)
    at MetadataDiscovery.findEntities (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/MetadataDiscovery.js:92:24)
    at MetadataDiscovery.discover (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/MetadataDiscovery.js:34:9)
    at MikroORM.discoverEntities (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/MikroORM.js:90:25)
    at Function.init (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/MikroORM.js:45:9)
2022-07-09 18:02:39 error: uncaughtException: Entity 'Collection' was not discovered, please make sure to provide it in 'entities' array when initializing the ORM (used in User.projects)
MetadataError: Entity 'Collection' was not discovered, please make sure to provide it in 'entities' array when initializing the ORM (used in User.projects)
    at Function.fromUnknownEntity (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/errors.js:148:16)
    at /Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/MetadataValidator.js:55:46
    at Array.forEach (<anonymous>)
    at /Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/MetadataValidator.js:53:67
    at Array.forEach (<anonymous>)
    at MetadataValidator.validateDiscovered (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/MetadataValidator.js:53:20)
    at MetadataDiscovery.findEntities (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/MetadataDiscovery.js:92:24)
    at MetadataDiscovery.discover (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/MetadataDiscovery.js:34:9)
    at MikroORM.discoverEntities (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/MikroORM.js:90:25)
    at Function.init (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/MikroORM.js:45:9)
2022-07-09 18:05:46 error: uncaughtException: Entity 'Collection' was not discovered, please make sure to provide it in 'entities' array when initializing the ORM (used in User.projects)
MetadataError: Entity 'Collection' was not discovered, please make sure to provide it in 'entities' array when initializing the ORM (used in User.projects)
    at Function.fromUnknownEntity (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/errors.js:148:16)
    at /Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/MetadataValidator.js:55:46
    at Array.forEach (<anonymous>)
    at /Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/MetadataValidator.js:53:67
    at Array.forEach (<anonymous>)
    at MetadataValidator.validateDiscovered (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/MetadataValidator.js:53:20)
    at MetadataDiscovery.findEntities (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/MetadataDiscovery.js:92:24)
    at MetadataDiscovery.discover (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/MetadataDiscovery.js:34:9)
    at MikroORM.discoverEntities (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/MikroORM.js:90:25)
    at Function.init (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/MikroORM.js:45:9)
2022-07-09 18:06:12 error: uncaughtException: Please provide either 'type' or 'entity' attribute in User.projects
Error: Please provide either 'type' or 'entity' attribute in User.projects
    at ReflectMetadataProvider.initPropertyType (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/ReflectMetadataProvider.js:14:19)
    at /Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/ReflectMetadataProvider.js:9:54
    at ReflectMetadataProvider.initProperties (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/MetadataProvider.js:34:23)
    at ReflectMetadataProvider.loadEntityMetadata (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/ReflectMetadataProvider.js:9:9)
    at MetadataDiscovery.discoverEntity (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/MetadataDiscovery.js:224:13)
    at MetadataDiscovery.discoverReferences (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/MetadataDiscovery.js:163:13)
    at MetadataDiscovery.findEntities (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/MetadataDiscovery.js:90:9)
    at MetadataDiscovery.discover (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/MetadataDiscovery.js:34:9)
    at MikroORM.discoverEntities (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/MikroORM.js:90:25)
    at Function.init (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/MikroORM.js:45:9)
2022-07-09 18:06:16 error: uncaughtException: Please provide either 'type' or 'entity' attribute in User.projects
Error: Please provide either 'type' or 'entity' attribute in User.projects
    at ReflectMetadataProvider.initPropertyType (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/ReflectMetadataProvider.js:14:19)
    at /Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/ReflectMetadataProvider.js:9:54
    at ReflectMetadataProvider.initProperties (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/MetadataProvider.js:34:23)
    at ReflectMetadataProvider.loadEntityMetadata (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/ReflectMetadataProvider.js:9:9)
    at MetadataDiscovery.discoverEntity (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/MetadataDiscovery.js:224:13)
    at MetadataDiscovery.discoverReferences (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/MetadataDiscovery.js:163:13)
    at MetadataDiscovery.findEntities (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/MetadataDiscovery.js:90:9)
    at MetadataDiscovery.discover (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/MetadataDiscovery.js:34:9)
    at MikroORM.discoverEntities (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/MikroORM.js:90:25)
    at Function.init (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/MikroORM.js:45:9)
2022-07-09 18:07:15 error: uncaughtException: Please provide either 'type' or 'entity' attribute in User.projects
Error: Please provide either 'type' or 'entity' attribute in User.projects
    at ReflectMetadataProvider.initPropertyType (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/ReflectMetadataProvider.js:14:19)
    at /Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/ReflectMetadataProvider.js:9:54
    at ReflectMetadataProvider.initProperties (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/MetadataProvider.js:34:23)
    at ReflectMetadataProvider.loadEntityMetadata (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/ReflectMetadataProvider.js:9:9)
    at MetadataDiscovery.discoverEntity (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/MetadataDiscovery.js:224:13)
    at MetadataDiscovery.discoverReferences (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/MetadataDiscovery.js:163:13)
    at MetadataDiscovery.findEntities (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/MetadataDiscovery.js:90:9)
    at MetadataDiscovery.discover (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/MetadataDiscovery.js:34:9)
    at MikroORM.discoverEntities (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/MikroORM.js:90:25)
    at Function.init (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/MikroORM.js:45:9)
2022-07-09 18:09:09 error: uncaughtException: Entity 'project' was not discovered, please make sure to provide it in 'entities' array when initializing the ORM (used in User.projects)
MetadataError: Entity 'project' was not discovered, please make sure to provide it in 'entities' array when initializing the ORM (used in User.projects)
    at Function.fromUnknownEntity (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/errors.js:148:16)
    at /Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/MetadataValidator.js:55:46
    at Array.forEach (<anonymous>)
    at /Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/MetadataValidator.js:53:67
    at Array.forEach (<anonymous>)
    at MetadataValidator.validateDiscovered (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/MetadataValidator.js:53:20)
    at MetadataDiscovery.findEntities (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/MetadataDiscovery.js:92:24)
    at MetadataDiscovery.discover (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/MetadataDiscovery.js:34:9)
    at MikroORM.discoverEntities (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/MikroORM.js:90:25)
    at Function.init (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/MikroORM.js:45:9)
2022-07-09 20:51:31 error: [GET] /api/v1/user >> StatusCode:: 500, Message:: select "u0".* from "user" as "u0" - relation "user" does not exist
2022-07-09 20:51:37 error: [GET] /api/v1/user/33 >> StatusCode:: 500, Message:: select "u0".* from "user" as "u0" where "u0"."id" = 33 limit 1 - relation "user" does not exist
2022-07-09 20:53:17 error: [GET] /api/v1/user >> StatusCode:: 500, Message:: select "u0".* from "user" as "u0" - relation "user" does not exist
2022-07-09 20:54:17 error: [GET] /api/v1/user >> StatusCode:: 500, Message:: select "u0".* from "user" as "u0" - relation "user" does not exist
2022-07-09 20:55:52 error: [GET] /api/v1/user >> StatusCode:: 500, Message:: select "u0".* from "user" as "u0" - relation "user" does not exist
2022-07-09 20:57:37 error: [GET] /api/v1/user >> StatusCode:: 500, Message:: select "u0".* from "user" as "u0" - relation "user" does not exist
2022-07-09 20:58:07 error: [GET] /api/v1/user >> StatusCode:: 500, Message:: select "u0".* from "user" as "u0" - relation "user" does not exist
2022-07-09 21:00:11 error: [POST] /api/v1/user >> StatusCode:: 500, Message:: select "u0".* from "user" as "u0" where "u0"."email" = 'jonsnow@gmail.com' limit 1 - relation "user" does not exist
2022-07-09 21:18:02 error: uncaughtException: Please provide either 'type' or 'entity' attribute in User.projects
Error: Please provide either 'type' or 'entity' attribute in User.projects
    at ReflectMetadataProvider.initPropertyType (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/ReflectMetadataProvider.js:14:19)
    at /Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/ReflectMetadataProvider.js:9:54
    at ReflectMetadataProvider.initProperties (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/MetadataProvider.js:34:23)
    at ReflectMetadataProvider.loadEntityMetadata (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/ReflectMetadataProvider.js:9:9)
    at MetadataDiscovery.discoverEntity (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/MetadataDiscovery.js:224:13)
    at MetadataDiscovery.discoverReferences (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/MetadataDiscovery.js:163:13)
    at MetadataDiscovery.findEntities (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/MetadataDiscovery.js:90:9)
    at MetadataDiscovery.discover (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/MetadataDiscovery.js:34:9)
    at MikroORM.discoverEntities (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/MikroORM.js:90:25)
    at Function.init (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/MikroORM.js:45:9)
2022-07-09 21:19:09 error: uncaughtException: Please provide either 'type' or 'entity' attribute in User.projects
Error: Please provide either 'type' or 'entity' attribute in User.projects
    at ReflectMetadataProvider.initPropertyType (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/ReflectMetadataProvider.js:14:19)
    at /Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/ReflectMetadataProvider.js:9:54
    at ReflectMetadataProvider.initProperties (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/MetadataProvider.js:34:23)
    at ReflectMetadataProvider.loadEntityMetadata (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/ReflectMetadataProvider.js:9:9)
    at MetadataDiscovery.discoverEntity (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/MetadataDiscovery.js:224:13)
    at MetadataDiscovery.discoverReferences (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/MetadataDiscovery.js:163:13)
    at MetadataDiscovery.findEntities (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/MetadataDiscovery.js:90:9)
    at MetadataDiscovery.discover (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/MetadataDiscovery.js:34:9)
    at MikroORM.discoverEntities (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/MikroORM.js:90:25)
    at Function.init (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/MikroORM.js:45:9)
2022-07-09 21:21:31 error: [GET] /api/v1/project >> StatusCode:: 500, Message:: Cannot read properties of undefined (reading 'find')
2022-07-09 21:22:56 error: [GET] /api/v1/project >> StatusCode:: 500, Message:: Cannot read properties of undefined (reading 'findAll')
2022-07-09 21:23:32 error: [GET] /api/v1/project >> StatusCode:: 500, Message:: Cannot read properties of undefined (reading 'findAll')
2022-07-09 21:24:07 error: [GET] /api/v1/project >> StatusCode:: 500, Message:: Cannot read properties of undefined (reading 'findAll')
2022-07-09 21:25:00 error: [GET] /api/v1/project >> StatusCode:: 404, Message:: No projects found
2022-07-09 21:43:25 error: [POST] /api/v1/project >> StatusCode:: 500, Message:: Cannot read properties of undefined (reading 'projectService')
2022-07-09 23:35:23 error: [POST] /api/v1/project >> StatusCode:: 500, Message:: Value for User.firstName is required, 'undefined' found
entity: User {
  createdAt: 2022-07-10T06:35:23.430Z,
  updatedAt: 2022-07-10T06:35:23.430Z,
  projects: Collection<Project> { initialized: true, dirty: false },
  id: 1
}
2022-07-09 23:35:32 error: [POST] /api/v1/project >> StatusCode:: 500, Message:: Value for User.firstName is required, 'undefined' found
entity: User {
  createdAt: 2022-07-10T06:35:23.430Z,
  updatedAt: 2022-07-10T06:35:23.430Z,
  projects: Collection<Project> { initialized: true, dirty: false },
  id: 1
}
